generators/server/templates/src/main/resources/config/_application.yml

```diff
@@ -50,21 +50,18 @@ eureka:
         status-page-url-path: ${management.context-path}/info
         health-check-url-path: ${management.context-path}/health
         metadata-map:
-            zone: primary # This is needed for the load balancer
             profile: ${spring.profiles.active}
             version: ${info.project.version}
 ribbon:
     eureka:
         enabled: true
+    NFLoadBalancerRuleClassName: com.netflix.loadbalancer.WeightedResponseTimeRule
 <%_ } _%>
 <%_ if (applicationType === 'gateway') { _%>
-# See http://cloud.spring.io/spring-cloud-netflix/spring-cloud-netflix.html
-zuul: # those values must be configured depending on the application specific needs
+zuul:
     host:
-        max-total-connections: 1000
```

Comment:
don't change those 3 values, I modified those on purpose, those have changed (and we tested them with higher values than the default)